Source/ACE/Entity/DebugObject.cs

```diff
@@ -77,6 +77,7 @@ public DebugObject(ObjectGuid guid,  BaseAceObject baseAceObject)
                 // Creating from a pcap of the weenie - this will be set by the loot generation factory. Og II
                 this.PhysicsData.PhysicsDescriptionFlag &= ~PhysicsDescriptionFlag.Parent;
                 this.GameData.ValidLocations = (EquipMask)baseAceObject.ValidLocations;
+                this.GameData.IconOverlay = (ushort)baseAceObject.IconOverlayId;
```

Comment:
If I remember correctly we went the way of changing the type to int in the other cases were this was a problem as well, i.e. textures, palettes, etc.